time        stamp
uint16      port
TopicInfo[] topics